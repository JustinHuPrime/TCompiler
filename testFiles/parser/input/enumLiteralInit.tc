module foo;

enum bar {
  A,
  B = 5,
  C,
};