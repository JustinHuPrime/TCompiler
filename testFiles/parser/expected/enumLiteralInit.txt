testFiles/parser/enumLiteralInit.tc (code):
FILE(1, 1, STAB(ENTRY(bar, ENUM(testFiles/parser/enumLiteralInit.tc, 3, 1, CONSTANT(A, 0), CONSTANT(B, 5), CONSTANT(C, 6)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), ENUMDECL(3, 1, ID(3, 6, bar, REFERENCES(testFiles/parser/enumLiteralInit.tc, 3, 1)), ID(4, 3, A, REFERENCES(testFiles/parser/enumLiteralInit.tc, 4, 3)), ID(5, 3, B, REFERENCES(testFiles/parser/enumLiteralInit.tc, 5, 3)), ID(6, 3, C, REFERENCES(testFiles/parser/enumLiteralInit.tc, 6, 3)), (null), LITERAL(5, 7, UBYTE(5)), (null)))
